virtwho/virt/ahv/ahv_interface.py

```diff
@@ -596,20 +596,27 @@ def get_hosts_uuid(self, version):
                         "host object: %s" % (host_entity)
                     )
 
-    def get_host_cluster_uuid(self, host_info):
+    def get_host_cluster_name(self, host_info, cluster_ids):
         """
-        Returns host's cluster UUID.
+        Returns host's cluster name.
         Args:
             host_info (dict): Host info dict.
+            cluster_ids: Map of UUID to name for clusters
         Returns:
-            host_cluster_uuid (uuid): host's cluster uuid.
+            host_cluster_name: host's cluster name.
         """
+        this_uuid = None
         if 'cluster_uuid' in host_info:
-            return host_info['cluster_uuid']
+            this_uuid = host_info['cluster_uuid']
         elif 'cluster_reference' in host_info:
-            return host_info['cluster_reference']['uuid']
+            this_uuid = host_info['cluster_reference']['uuid']
 
-    def get_ahv_cluster_uuids(self, version):
+        for uuid, name in cluster_ids:
+            if this_uuid == uuid:
+                return name
+        return this_uuid
+
+    def get_ahv_cluster_uuid_map(self, version):
```

Comment:
I thinks that doc string should be changed according change in the code. E.g the method returns list of tuples.